ftplugin/javascript/splitjoin.vim

```diff
@@ -10,6 +12,8 @@ endif
 
 if !exists('b:splitjoin_join_callbacks')
   let b:splitjoin_join_callbacks = [
+        \ 'sj#html#JoinAttributes',
+        \ 'sj#html#JoinTags',
```

Comment:
I think you don't need this here. I haven't really worked with JSX, so I might be missing something, but if you have the ftplugin for the "jsx" filetype, shouldn't that be enough? If the filetype is something like "html.jsx", you can simply remove the `if !exists(...)` from the jsx ftplugin stuff and it will take precedence over the html definitions.